components/Dialogs/examples/DialogsAlertViewController.m

```diff
@@ -60,16 +61,22 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
   }
 }
 
+- (void)themeAlertController:(MDCAlertController *)alertController {
```

Comment:
is it worth possibly instantiating the colorScheme and typographyScheme once, and having them be accessible throughout this class rather than re-instantiating? Normally I think it's ok to leave as is, but it looks like they are initialized quite a bit.

Reply:
Done.